engine/pkg/meta/model/config.go

```diff
@@ -120,6 +122,24 @@ func GenerateDSNByParams(storeConf *StoreConfig, pairs map[string]string) string
 	if storeConf.Password != "" {
 		dsnCfg.Passwd = storeConf.Password
 	}
+
+	if storeConf.Security != nil {
+		cfg, err := util.NewTLSConfig(
+			util.WithCAPath(storeConf.Security.CAPath),
+			util.WithCertAndKeyPath(storeConf.Security.CertPath, storeConf.Security.KeyPath),
+			util.WithVerifyCommonName(storeConf.Security.CertAllowedCN))
+		if err != nil {
+			return fmt.Sprintf("%v", err)
```

Comment:
this is hard to debug (for example, failed to access CAPath) and `GenerateDSNByParams` is no longer a pure function. How about moving `CreateSchemaIfNotExists` under the sqlkv package? @maxshuang

Reply:
`sqlkv` is the pkg in an `internal`, it can't used by `server.go`?

Reply:
PTAL. I add an return error for `GenerateDSNByParams `.

Reply:
LGTM